drizzle-kit/src/sqlgenerator.ts

```diff
@@ -160,13 +160,11 @@ class PgCreateRoleConvertor extends Convertor {
 		return statement.type === 'create_role' && dialect === 'postgresql';
 	}
 	override convert(statement: JsonCreateRoleStatement): string | string[] {
-		return `CREATE ROLE "${statement.name}"${
-			statement.values.createDb || statement.values.createRole || !statement.values.inherit
-				? ` WITH${statement.values.createDb ? ' CREATEDB' : ''}${statement.values.createRole ? ' CREATEROLE' : ''}${
-					statement.values.inherit ? '' : ' NOINHERIT'
-				}`
-				: ''
-		};`;
+		return `CREATE ROLE "${statement.name}"${statement.values.createDb || statement.values.createRole || !statement.values.inherit
+			? ` WITH${statement.values.createDb ? ' CREATEDB' : ''}${statement.values.createRole ? ' CREATEROLE' : ''}${statement.values.inherit ? '' : ' NOINHERIT'
+			}`
+			: ''
+			};`;
```

Comment:
Same here, these don't look actually changed. Could lighten the diff if we removed all these non-changes.

Reply:
Yeah, I have reverted this unnecessary change to make diff clear.